src/sql/workbench/contrib/welcome/gettingStarted/browser/initialSetupWizard.css

```diff
@@ -0,0 +1,295 @@
+	}
+}
+
+.ads-initial-setup-wizard-popup
+	.ads-initial-setup-wizard-popup-content-container {
+	padding: 22px 0px 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	height: 100%;
+}
+
+.ads-initial-setup-wizard-popup
+	.ads-initial-setup-wizard-popup-content-container
+	p {
+	font-size: 1em;
+}
+
+.ads-initial-setup-wizard-popup {
+	position: absolute;
+	width: 720px;
+	max-height: 592px;
+	margin: auto;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: center;
+	opacity: 0;
+}
+
+.ads-initial-setup-wizard-add-account.ads-initial-setup-wizard-interaction-container {
+	max-width: 109px;
+}
+
+.ads-initial-setup-wizard .ads-initial-setup-wizard-step {
+	font-size: 14px;
+	font-weight: 600;
+}
+
+.ads-initial-setup-wizard-btn-container {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.ads-initial-setup-wizard-previous-btn-container,
+.ads-initial-setup-wizard-next-btn-container {
+	margin: 0 4px;
+	width: 77px;
+}
+
+.ads-initial-setup-wizard-nav-container {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 15px 24px;
+	border-top: 1px solid;
+}
+
+.ads-initial-setup-wizard-btn-container p {
+	font-size: 1.5em;
+	margin: 0;
+}
+
+.ads-initial-setup-wizard-btn-container:first-of-type(p) {
+	margin-bottom: 20px;
+}
+
+.ads-initial-setup-wizard-popup .flex.column {
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+}
+
+.ads-initial-setup-wizard-popup h1,
+.ads-initial-setup-wizard-popup p {
+	font-family: Segoe UI, Frutiger, Frutiger Linotype, Dejavu Sans,
+		Helvetica Neue, Arial, sans-serif;
+	font-weight: 400;
+}
+
+.ads-initial-setup-wizard-popup h1 {
+	margin: 0;
+	font-size: 18px;
+	font-weight: 700;
+	line-height: 24px;
+}
+
+.ads-initial-setup-wizard-popup {
+	display: none;
+	z-index: 1;
+}
+
+.ads-initial-setup-wizard-popup.ads-initial-setup-wizard-show {
+	display: block;
+	border-radius: 2px;
+	opacity: 1;
+}
+
+.ads-initial-setup-wizard-popup .ads-initial-setup-wizard-img {
+	width: 720px;
+	min-height: 264px;
+}
+
+.ads-initial-setup-wizard-popup .ads-initial-setup-wizard-docs-link {
+	align-self: center;
+	justify-self: flex-start;
+	margin-right: auto;
+	text-decoration: none;
+	width: 77px;
+}
+
+.ads-initial-setup-wizard-popup .icon-link {
+	display: inline-block;
+	width: 11px;
+	height: 11px;
+	margin-left: 4px;
+	-webkit-mask: url(../../media/link_icon.svg) no-repeat;
+	-webkit-mask-size: 11px 11px;
+	mask: url(../../media/link_icon.svg) no-repeat;
+	mask-size: 11px 11px;
+	height: 11px;
+}
+
+.ads-initial-setup-wizard-btn {
+	box-sizing: border-box;
+	width: 100%;
+	height: 34px;
+	padding-top: 1px;
+	font-size: 1.08em;
+	line-height: 29px;
+	border-radius: 2px;
+	display: flex;
+	align-content: center;
+	justify-content: center;
+	text-align: center;
+	text-decoration: none;
+	margin: 10px 0;
+}
+
+.activitybar
```

Comment:
Scope this properly under the setup wizard - we shouldn't be affecting other bars in the product